### PR TITLE
Switch bed file start and end positions if start > end

### PR DESCRIPTION
**Pull request by pkerpedjiev**
The specification for bed files is not completely clear on whether start position should be less than end position and some people may have be greater in the case of minus strand. This PR checks to see if the start position is greater than the end and switches them if not.

### CODE REVIEW

File: clodius/cli/aggregate.py [hunk 1]
```diff
@@ -451,8 +451,10 @@ def line_to_np_array(line):
             importance = int(line[int(importance_column)-1])
 
         if stop < start:
-            print("WARNING: stop < start:", line, file=sys.stderr)
-            return
+            # print("WARNING: stop < start:", line, file=sys.stderr)
```
Comment (nvictus) [inline clodius/cli/aggregate.py]: If at least one of these is found, it should definitely print a warning (if not throw an error and force the user to fix the input themselves). In most cases, having stop < start won't be intentional.

File: clodius/cli/aggregate.py [hunk 2]
```diff
@@ -451,8 +451,10 @@ def line_to_np_array(line):
...
+            temp = start
```
Comment (nvictus) [inline clodius/cli/aggregate.py]: Not Pythonic.
```suggestion
            start, stop = stop, start
```

File: clodius/cli/aggregate.py [hunk 3]
```diff
@@ -451,8 +451,10 @@ def line_to_np_array(line):
...
+            start = stop
```
Comment (nvictus) [inline clodius/cli/aggregate.py]:
```suggestion
```

File: clodius/cli/aggregate.py [hunk 4]
```diff
@@ -451,8 +451,10 @@ def line_to_np_array(line):
...
+            stop = temp
```
Comment (nvictus) [inline clodius/cli/aggregate.py]:
```suggestion
```

**Comment by pkerpedjiev** [comment]
I just realized that this PR also removes clodius fast. I must have had some staged changes that were included here. For the better, I think.

**Pull request merged**